flight/core/Dispatcher.php

```diff
@@ -23,41 +24,54 @@ class Dispatcher
 {
     public const FILTER_BEFORE = 'before';
     public const FILTER_AFTER = 'after';
-    private const FILTER_TYPES = [self::FILTER_BEFORE, self::FILTER_AFTER];
 
-    /** @var mixed $containerException Exception message if thrown by setting the container as a callable method */
-    protected $containerException = null;
+    /** Exception message if thrown by setting the container as a callable method. */
+    protected ?Exception $containerException = null;
 
-    /** @var ?Engine $engine Engine instance */
+    /** @var ?Engine $engine Engine instance. */
     protected ?Engine $engine = null;
 
-    /** @var array<string, Closure(): (void|mixed)> Mapped events. */
+    /** @var array<string, callable(): (void|mixed)> Mapped events. */
     protected array $events = [];
 
     /**
      * Method filters.
      *
-     * @var array<string, array<'before'|'after', array<int, Closure(array<int, mixed> &$params, mixed &$output): (void|false)>>>
+     * @var array<string, array<'before'|'after', array<int, callable(array<int, mixed> &$params, mixed &$output): (void|false)>>>
      */
     protected array $filters = [];
 
     /**
      * This is a container for the dependency injection.
      *
-     * @var callable|object|null
+     * @var null|ContainerInterface|(callable(string $classString, array<int, mixed> $params): (null|object))
```

Comment:
Do you still need to correct this comment?

Reply:
yeah, before it was callable(... array) and phpstan was complaining that I told him array of what. Well, since they are arguments of a constructor, well, mixed